client/src/components/Users/Auth/SignUpUserAndNonprofit/SignUpUserAndNonprofit.tsx

```diff
@@ -604,7 +674,7 @@ export const SignUpUserAndNonprofit = () => {
                       variant="contained"
                       disabled={activeStep === 0}
                       onClick={handleBack}
-                      sx={{ mr: 1 }}
+                      sx={{ mr: 1, display: `${activeStep === 0 && 'none'}` }}
```

Comment:
Will this still work now that the multi-step component has been removed?

Reply:
yep!